app/src/main/java/com/owncloud/android/ui/fragment/UnifiedSearchFragment.kt

```diff
@@ -251,6 +252,7 @@ class UnifiedSearchFragment : Fragment(), Injectable, UnifiedSearchListInterface
     }
 
     override fun onQueryTextSubmit(query: String): Boolean {
+        KeyboardUtils.hideKeyboardFrom(requireContext(), binding.root)
```

Comment:
@tobiasKaminsky This logic will hide the keyboard every time whenever user do search.
Let me know if this also has to be up-streamed.